core/sdk/src/main/java/com/mongodb/stitch/core/internal/net/StitchRequest.java

```diff
@@ -19,6 +19,7 @@
 import java.util.Arrays;
```

Comment:
This file can be reverted for the sake of minimizing code change. We can driveby format later.

Reply:
done